python/setup.py

```diff
@@ -2,7 +2,7 @@
 
 
 setuptools.setup(
-    name='energuide-etl',
+    name='energuide',
```

Comment:
I thought this might be causing some troubles locally, but it turns out not. Better to do anyway I think.